tests/test_templater.py

```diff
@@ -291,3 +292,26 @@ def test_set_template_verbosity():
     outcome = ''
     # Check that only the correct messages were logged
     assert result == outcome
+
+def test_log_passing():
+    """Unit test for checking application of the logger object passing"""
+
+    input_file = os.path.join(uwtools_file_base, "fixtures/nml.IN")
+    logfile = os.path.join(os.path.dirname(templater.__file__), "templater.log")
+    ref = "Only the templater logger would see this"
+
+    args = [
+        '-i', input_file,
+        '--dry_run',
+        '-v'
+        ]
+
+    print("Test templater with logger decorator")
+    templater.set_template(args)
+    print("Test calling templater logging object")
+    log = Logger(name='test_templater.uwtools.templater').get_logger()
```

Comment:
Currently some issue with this implementation, testing now

Reply:
I suspect the issue is that logger.py is overwriting the logger object after instantiating it. I would recommend changing back to logging.getLogger() for now and opening an issue to add the "if _name in logging.root.manager.loggerDict" check to logger to bypass the level and handlers if the object already exists

Reply:
Reverted and created issue #190